build.sbt

```diff
@@ -20,7 +20,25 @@ val Scala213 = "2.13.10"
 ThisBuild / crossScalaVersions := Seq("2.12.15", Scala213, "3.2.0")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
-lazy val root = tlCrossRootProject.aggregate(trace4Cats)
+lazy val root = tlCrossRootProject.aggregate(catsEffect, trace4Cats)
+
+lazy val catsEffect = project
+  .in(file("cats-effect"))
+  .settings(
+    name := "prometheus4cats-contrib-cats-effect",
+    libraryDependencies ++= Seq(
+      "com.permutive" %% "prometheus4cats" % "1.0.0-RC3",
+      "org.typelevel" %% "cats-effect" % "3.3.14",
```

Comment:
Should these dependencies be extracted somewhere to enforce that each module depends on the same version?

Reply:
Ah yeah, prom4cats probably should, but CE is only needed on this module